src/main/java/seedu/address/model/notes/exceptions/NoteNotFoundException.java

```diff
@@ -4,4 +4,7 @@
  * Signals that the operation is unable to find the specified note.
  */
 public class NoteNotFoundException extends RuntimeException {
+    public NoteNotFoundException() {
+        super("Note is not found");
```

Comment:
Maybe the error message could be more specific, such as the note index is more than the size of note list?